### PR TITLE
Added proposed API to readme

### PR DESCRIPTION
**Pull request by n0x3u5**
@siawo Have a look at the proposed API. Let me know what you think. If all seems okay, we can go ahead and merge this.

### CODE REVIEW

**Comment by n0x3u5** [comment]
PR for #3 .

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
-# fusionbook
+# FusionBook
 Develop and test FusionCharts components in isolation
+
```
Comment (siawo) [inline README.md:3]: We need to add a parser that understands the type and name of the component.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
...
+legendItemStory.addChapter(
+  'with background',
+  () => story.attachChild(LegendItem).configure({ showBackground: true })
+);
+
+// Creates a story named "Div Element"
+const divStory = new Story('Div Element');
+
+// Adds a "with 50% opacity" chapter to the story
+legendItemStory.addChapter(
```
Comment (siawo) [inline README.md]: legendItem should be divStory

**Comment by n0x3u5** [comment]
@siawo Suggested changes made. Have a look.

**Review by siawo — APPROVED** [review]

**Pull request merged**